src/scripts/pagination-next-page.ts

```diff
@@ -0,0 +1,23 @@
+import { renderGarageFields } from './render-app';
+
+function nextPage(event: Event): void {
```

Comment:
I can see only one different between nextPage and prevPage. In my opinion it might be one function. For example change page and you can check target and do what you need +1 or -1.